.gitignore

```diff
@@ -1,4 +1,5 @@
 .DS_Store
+.idea
```

Comment:
is this something your IDE uses?

Reply:
Yes, JetBrains creates a `.idea` directory. I didn't have issues with spacestation because this line is already in the `.gitignore`.